src/index.js

```diff
@@ -6,12 +6,26 @@ function removeNaked (): string {
   return domain.indexOf('www.') === 0 ? domain.substring(4) : domain
 }
 
-function findOrCreate (name: string, expires: number, data: ?string): string {
+function findOrCreate (
+  name: string,
+  expires: number,
+  data: ?string,
+  path: ?string,
+  secure: ?boolean
+): string {
   const value = cookies.get(name)
   if (value) {
     return value
   }
 
+  let attribute = { expires, secure }
+  if (path) {
```

Comment:
js-cookie set to `/` as default, but if set `undefined` to path attribute, use access path. (For example, `/path`).